### PR TITLE
Publishing Flow for heyVR.io

### PR DESCRIPTION
**Pull request by JackJohansson**
This PR adds a documentation page on how to publish PlayCanvas games on heyVR.io. 

image

**PS:** The License Agreement file seems to have been deleted. Please provide a link to a new file if you'd like us to sign it before merging the PR, Thanks.

I confirm I have read the [contributing guidelines](https://github.com/playcanvas/developer.playcanvas.com/blob/main/.github/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+title: Hosting on heyVR.io
+sidebar_position: 7
+---
+
+heyVR.io is a publishing platform for WebXR games, giving you access to a global audience of hundreds of thousands of players across multiple devices.
+It offers a wide range of ready-to-use gameplay features to enhance your players' experience and enable you to start earning from your game through in-game content sales and advertisements.
+
+## Publishing Flow
+
+To publish your game on heyVR, You'll need to follow a few simple steps:
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:11]:
```suggestion
To publish your game on heyVR, you'll need to follow a few simple steps:
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+1. Create a developer account (assuming you don't have one yet)
+2. Upload your game
+3. Publish!
+
+You can optionally integrate they [heyVR SDK][5] into your game, which will boost player engagement and grant you access to useful features such as leaderboards, cloud saves, matchmaking and more.
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:17]:
```suggestion
You can optionally integrate the [heyVR SDK][5] into your game, which will boost player engagement and grant you access to useful features such as leaderboards, cloud saves, matchmaking and more.
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+title: Hosting on heyVR.io
+sidebar_position: 7
+---
+
+heyVR.io is a publishing platform for WebXR games, giving you access to a global audience of hundreds of thousands of players across multiple devices.
+It offers a wide range of ready-to-use gameplay features to enhance your players' experience and enable you to start earning from your game through in-game content sales and advertisements.
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:7]:
```suggestion
heyVR.io is a publishing platform for WebXR games, giving you access to a global audience of hundreds of thousands of players across multiple devices. It offers a wide range of ready-to-use gameplay features to enhance your players' experience and enable you to start earning from your game through in-game content sales and advertisements.
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+1. Create a developer account (assuming you don't have one yet)
+2. Upload your game
+3. Publish!
+
+You can optionally integrate they [heyVR SDK][5] into your game, which will boost player engagement and grant you access to useful features such as leaderboards, cloud saves, matchmaking and more.
+
+### 1. Create a developer account
+
+Simply head over to the heyVR's [developer area][2] and register your developer account free of charge. 
+After registration, you'll be able to create/manage your games. To read more, check out the documentations (link below).
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:22]:
```suggestion
Simply head over to the heyVR's [developer area][2] and register your developer account free of charge. After registration, you'll be able to create/manage your games. To read more, check out the [documentation](https://docs.heyvr.io).
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+### 2. Upload your game
+
+Once you have created your developer account, you'll need to download a web build of your application.
+The steps for this process are explained in details on the [Self-hosting][1] documentation page. 
+Simply follow the steps for **Self-hosting on your own server** steps to acquire a **zip** file.
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:28]:
```suggestion
Once you have created your developer account, you'll need to download a web build of your application. The steps for this process are explained in details on the [Self-hosting](/user-manual/publishing/web/self-hosting/#self-hosting-on-your-own-server) documentation page. Simply follow the steps for **Self-hosting on your own server** steps to acquire a **zip** file.
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+Once you have the zip build, you can follow the step-by-step guide on [publishing a game][4] on heyVR.
+The guide includes all the steps, requirements and common questions regarding the publishing process.
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:31]:
```suggestion
Once you have the zip build, you can follow the step-by-step guide on [publishing a game](https://docs.heyvr.io/en/developer-area/publish-a-game) on heyVR. The guide includes all the steps, requirements and common questions regarding the publishing process.
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+### 3. Publish!
+
+That's it! By now you should be able to see, view and test your game on heyVR. 
+Once you're ready to make it publicly accessible, you can submit your game for review. 
+The review process is to ensure that your game aligns with the [Publishing Guidelines][8] and is fully functional.
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:37]:
```suggestion
That's it! By now you should be able to see, view and test your game on heyVR. Once you're ready to make it publicly accessible, you can submit your game for review. The review process is to ensure that your game aligns with the [Publishing Guidelines](https://heyvr.io/hc/legal/heyvr-io-publishing-policies) and is fully functional.
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+### 4. SDK Implementation (optional)
+
+To further enrich the user experience and boost engagement to your game, you can use the [heyVR SDK][5] to implement additional features in your game.
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:41]:
```suggestion
To further enrich the user experience and boost engagement to your game, you can use the [heyVR SDK](https://docs.heyvr.io/en/game-development/sdk) to implement additional features in your game. 
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+To further enrich the user experience and boost engagement to your game, you can use the [heyVR SDK][5] to implement additional features in your game. 
+
+The SDK offers the following features and more:
+
+- User Management
+- Leaderboards
+- Cloud Saves
+- In-Game Purchases (both free & paid)
+- In-Game Advertisement
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:49]:
```suggestion
- In-Game Advertisements
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+- Multiplayer
+
+A comprehensive guide on how to implement these features into your game is provided on heyVR's [Technical Documentation][5].
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:52]:
```suggestion
A comprehensive guide on how to implement these features into your game is provided on heyVR's [Technical Documentation](https://docs.heyvr.io/en/game-development/sdk).
```

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+- [heyVR Support (Discord)][7]
+
+[1]: /user-manual/publishing/web/self-hosting/#self-hosting-on-your-own-server
+[2]: https://developer.heyvr.io
+[3]: https://docs.heyvr.io
+[4]: https://docs.heyvr.io/en/developer-area/publish-a-game
+[5]: https://docs.heyvr.io/en/game-development/sdk
+[6]: https://heyvr.io
+[7]: https://heyvr.io/discord
+[8]: https://heyvr.io/hc/legal/heyvr-io-publishing-policies
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:68]:
```suggestion
- [heyVR.io](https://heyvr.io)
- [Developer Area](https://developer.heyvr.io)
- [heyVR Documentation](https://docs.heyvr.io)
- [heyVR Support (Discord)](https://heyvr.io/discord)
```

**Review by willeastcott — APPROVED** [review]

**Comment by willeastcott** [comment]
Approving with a cluster of suggestions.

**Comment by JackJohansson** [comment]
@willeastcott Thanks for the edits, just wanted to check if [this section](https://github.com/playcanvas/developer.playcanvas.com/blob/main/.github/CONTRIBUTING.md#hyperlinks) of the guideline is deprecated?

**Comment by willeastcott** [comment]
Yeah, I think we are gradually moving away from placing all links at the bottom. I should update the contributing guide.

**Review by JackJohansson — COMMENTED** [review]
All look good. Sorry for the delay. Do I need to commit them individually or would you be able to merge after the review?

**Review by willeastcott — APPROVED** [review]
Thanks for the contribution!

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+- In-Game Advertisements
+- Multiplayer
+
+A comprehensive guide on how to implement these features into your game is provided on heyVR's [Technical Documentation](https://docs.heyvr.io/en/game-development/sdk).
+
+## Useful Links
+
+- [Developer Area](https://developer.heyvr.io)
+- [heyVR Documentation](https://docs.heyvr.io)
+- [heyVR Support (Discord)](https://heyvr.io/discord)
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:51]: Add a carriage return here to end the file.

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+
+Simply head over to the heyVR's [developer area](https://developer.heyvr.io) and register your developer account free of charge. After registration, you'll be able to create/manage your games. To read more, check out the [documentation](https://docs.heyvr.io).
+
+### 2. Upload your game
+
+Once you have created your developer account, you'll need to download a web build of your application. The steps for this process are explained in details on the [Self-hosting](/user-manual/publishing/web/self-hosting/#self-hosting-on-your-own-server) documentation page. Simply follow the steps for **Self-hosting on your own server** to acquire a **zip** file.
+
+Once you have the zip build, you can follow the step-by-step guide on [publishing a game](https://docs.heyvr.io/en/developer-area/publish-a-game) on heyVR. The guide includes all the steps, requirements and common questions regarding the publishing process.
+
+### 3. Publish!
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:28]: Please remove the `!`.

File: docs/user-manual/publishing/web/hosting-heyvr.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+
+That's it! By now you should be able to see, view and test your game on heyVR. Once you're ready to make it publicly accessible, you can submit your game for review. The review process is to ensure that your game aligns with the [Publishing Guidelines](https://heyvr.io/hc/legal/heyvr-io-publishing-policies) and is fully functional.
+
+### 4. SDK Implementation (optional)
+
+To further enrich the user experience and boost engagement to your game, you can use the [heyVR SDK](https://docs.heyvr.io/en/game-development/sdk) to implement additional features in your game.
```
Comment (willeastcott) [inline docs/user-manual/publishing/web/hosting-heyvr.md:34]: Please remove the blank space at the end of this line.

**Comment by willeastcott** [comment]
Please address the 3 linting errors in the markdown and I'll merge. Thanks!

**Comment by JackJohansson** [comment]
> Please address the 3 linting errors in the markdown and I'll merge. Thanks!

Sure. I've updated the docs to address the issues

**Comment by willeastcott** [comment]
Thanks. Your page is now live here:

https://developer.playcanvas.com/user-manual/publishing/web/hosting-heyvr/

**Pull request merged**